### ISSUE TITLE
Better editors choice and recommended applications

### ISSUE THREAD

**Issue by TingPing** [flathub/linux-store-frontend#69]
The goal of the home page should be to show the best Flathub has to offer and I am confident that applications like Steam are not that. I think we should set some ground rules for what not to promote. This may be painful since the ones winning the popularity contest don't match my recommendations but I think it is for the best.

Editors choices should not:

- Use `extra-data`. These can fail to install sometimes which is a bad UX.
- Use a toolkit other than Gtk3 or Qt5. We want features like portals and theams working.
- Be complex applications like an IDE. These are where the friction of the sandbox is most visible and an unexpected UX for developers familiar with existing workflows to say the least.

**Comment by barthalion** [comment]
I agree with everything. We can revise proprietary rule if some closed-source upstream expresses interest in shipping their app on Flathub but that's not the case now.

**Comment by TingPing** [comment]
> If there are bugs with popular apps, we ought to work to resolve to fix them, not hide them away.

In a perfect world I agree. How do we solve Steam? Do you know the one or two folks working at Valve on their Linux port? Can you get them to dedicate time to fixing obscure bugs in an unsupported downstream package? Can you get them to get in contact with third party game developers publishing games to an unsupported downstream package?

It is a bad example of what Flatpak has to offer and cannot be fixed sorry. We can however not promote it like it is the best we can do.

**Comment by TingPing** [comment]
Just to prove my point: https://www.reddit.com/r/linux/comments/8aixg0/a_new_version_of_flathub_to_browse_flatpak_apps/

Both Steam and VSCode are the first thing complained about for the reasons I mentioned. These are not solvable problems and we shouldn't shove them in users faces. It makes Flatpak and Flathub look bad.

**Comment by TingPing** [comment]
> If you read the Ubuntu Insights blog most of the snap related posts are about popular proprietary software (Spotify, Skype, Slack, ...)

Spotify, Skype, and Slack have upstream developers supporting those packages. We do not.

I agree with your requirements overall.

> Once we agree on the editor's choice criteria, how do create the list of apps? In this issue and asking people to "like" the proposals? With Google Forms?

Not sure honestly. There are only a few people involved in this process but there are also quite a lot of apps to sift through..

**Comment by nedrichards** [comment]
This PR doesn't fix the issue, but it should mitigate some of the effects: https://github.com/flathub/linux-store-frontend/pull/133
Specifically I would like to remove two extra-data apps that I help to maintain and replace them with more integrated flatpak apps that are maintained by upstream.

**Comment by jgarciao** [comment]
With the help of Google Analytic and[ this script](https://github.com/flathub/linux-store-frontend/blob/master/scripts/get-editors-choice-report.sh) I've created a little report with a list the most downloaded apps (from the website) with extra info like the number of open issues at github.

The report is [here](https://docs.google.com/spreadsheets/d/1ZojkZc8t2fxM-mMgFIJM02b2CrQo2JiuKxA0Zl4Wvyg/edit?usp=sharing).

With this information, I've updated the editor's picks for apps and games. You can see the resuting picks in the [dev server](http://45.55.104.129/apps/collection/editors-choice-apps).

This is just a temporally list until a better system is found. In the meantime, could you take a look to see the picks?

**Comment by SISheogorath** [comment]
To be honest when we call it "Editor's choice" it shouldn't be the "Most downloaded" or "Most accessed". It should be an informed decision by a person or a group of people that is regularly working and may maintaining Apps on FlatHub. Otherwise we should rename it since it's not what it appears to be.

And we already have "Popular Apps". Which, from my perspective, should be pretty much what you created with that list.

**Comment by TingPing** [comment]
New list seems ok :+1:

**Comment by jgarciao** [comment]
Sorry, you mean the updated  editor's lists or the "popular games" proposal?

El dj., 27 de set. 2018, 18:56, TingPing <notifications@github.com> va
escriure:

**Comment by TingPing** [comment]
I meant the new list but that sounds good too. For low maintenance just a Popular apps with a blacklist would be fine.
